docs/resources/js/lovers.js

```diff
@@ -26,6 +26,12 @@
 **/
 
 var peopleLovePapa = [
+	{
+		link: "https://www.vertex.io.com",
```

Comment:
This link is borken! Could you please fix it?

Reply:
🤦 fixed